cachelib/allocator/tests/MMTypeTest.h

```diff
@@ -343,5 +348,28 @@ size_t MMTypeTest<MMType>::getListSize(const Container& c,
   }
   throw std::invalid_argument("LruType not existing");
 }
+
+// Verifies that using getEvictionIterator and withEvictionIterator
+// yields the same elements, in the same order.
+template <typename MMType>
+void MMTypeTest<MMType>::verifyIterationVariants(Container& c) {
+  std::vector<Node*> nodes;
+  for (auto iter = c.getEvictionIterator(); iter; ++iter) {
+    nodes.push_back(&(*iter));
+  }
+
+  size_t i = 0;
+  c.withEvictionIterator([&c, &nodes, &i](auto&& iter) {
```

Comment:
Don't have to capture `c` here since it is not used inside.

Reply:
Done.